part-07.mdx

```diff
@@ -0,0 +1,126 @@
+
+The settings tab at the top of your Netdata dashboard houses browser settings. 
+These settings do not affect the operation of your Netdata server/daemon.. 
+They take effect immediately and are permanently saved to browser 
+local storage (except the refresh on focus / always option).
+
+You can see the Performance, Synchronization, Visual, and Locale tabs on the dashboard settings modal. 
+
+### Performance
+
+This tab is a collection of options focused on the performance of the
+ web browser by tweaking the default dashboard behavior. 
+
+You can set when to refresh the charts, which dimensions to show and how you to handle hidden charts, and page scroll.
+
+### Synchronization
+
+This tab is a collection of options focused on how your metrics and charts synchronize. 
+
+You can choose the **parallel** or **sequential** chart refresh policy to use. 
+Selecting **parallel** allows you options as to how you prefer re-sync chart refreshes.
+
+You can also choose to sync or not sync hover selection on all chart, and pan and zoom on all charts.
+
+### Visual
+
+This tab focuses on the visuals of your dashboard.
+
+You can switch between the default dark theme and white. 
+This action and switching the **Help Me** button triggers a reload of the dashboard.
+
+You can also enable data padding and Bézier lines on charts.
+
+### Locale
+
+This tab houses options to scale and select units for time and temperature.
+
+You can also choose either to use browser local time or server time. 
+
+However, note that these settings are applied gradually as charts are updated. 
+To force them, refresh the dashboard now. 
+
+![dash-settings](https://user-images.githubusercontent.com/12263278/64901553-967f3880-d692-11e9-95ac-dd485d36535c.gif)
+
+## Check if there's an update to Netdata
+
+Netdata upon installation with [the one-liner](https://docs.netdata.cloud/packaging/installer/#one-line-installation) 
+comes with the auto-update option enabled by default.
+
+You can always check if there is an update available right from the Update tab of your Netdata dashboard.
+
+![nedata-update](https://user-images.githubusercontent.com/12263278/64876743-be957a00-d647-11e9-83dd-2f0a8df572cb.gif)
+
+You can see an update is available from the above short screen record.
+
+Netdata automatically updates itself, but if you choose to update it manually, there are [several well-documented methods](https://docs.netdata.cloud/packaging/installer/update/) to achieve that. 
+However, it is best practice for you to first go over the [changelog](https://docs.netdata.cloud/changelog/).
+
+## Export or import a snapshot
+
+Snapshot files include all the information of the dashboard, including the URL of the origin server, 
+its netdata unique ID, and chart data queries for the visible timeframe,
+so align, pan and zoom the charts as needed. 
+Snapshots are not in real-time. 
+
+Netdata can export and import dashboard snapshots. Any Netdata can import the snapshot of any other Netdata. 
```

Comment:
I feel like this should be the first sentence of the section, then follow up with what a snapshot contains.